examples/Backtesting/botorch_analytical.py

```diff
@@ -11,30 +9,36 @@
 # 2. [`discrete_space`](./../Searchspaces/discrete_space.md) for details on using a
 # BoTorch test function.
 
-### Necessary imports for this example
+### Imports
+
+import os
+import sys
+from pathlib import Path
 
-import matplotlib.pyplot as plt
 import numpy as np
-import seaborn as sns
 from botorch.test_functions import Rastrigin
 
 from baybe import Campaign
 from baybe.objective import Objective
 from baybe.parameters import NumericalDiscreteParameter
-from baybe.recommenders import RandomRecommender, SequentialGreedyRecommender
+from baybe.recommenders import RandomRecommender
 from baybe.searchspace import SearchSpace
 from baybe.simulation import simulate_scenarios
 from baybe.strategies import TwoPhaseStrategy
 from baybe.targets import NumericalTarget
 from baybe.utils.botorch_wrapper import botorch_function_wrapper
+from baybe.utils.plotting import create_plots
 
 ### Parameters for a full simulation loop
 
-# For the full simulation, we need to define some additional parameters.
-# These are the number of Monte Carlo runs and the number of experiments to be conducted per run.
+# For the full simulation, we need to define the number of Monte Carlo runs
+# and the number of experiments to be conducted per run.
+
+SMOKE_TEST = "SMOKE_TEST" in os.environ
 
-N_MC_ITERATIONS = 2
-N_DOE_ITERATIONS = 2
+N_MC_ITERATIONS = 2 if SMOKE_TEST else 30
+N_DOE_ITERATIONS = 2 if SMOKE_TEST else 15
+BATCH_SIZE = 1 if SMOKE_TEST else 3
```

Comment:
we need to make the palcement of these variable mroe consistent, imo there are two versions:

1) all constants at the top o file
2) directly before they are needed

currently eg `POINTS_PER_DIM` uses 2 but all others use 1

Reply:
My feeling is that 1. us more reasonable for the examples since it let's you easily and quickly change different settings for experimentation. Will adjust in this example.

Reply:
Have adjusted, hence resolved.